src/libraries/System.DirectoryServices/src/System/DirectoryServices/ExternDll.cs

```diff
@@ -6,45 +6,7 @@ namespace System
     internal static class ExternDll
     {
         public const string Activeds = "activeds.dll";
-        public const string Advapi32 = "advapi32.dll";
-        public const string Comctl32 = "comctl32.dll";
-        public const string Comdlg32 = "comdlg32.dll";
-        public const string Gdi32 = "gdi32.dll";
-        public const string Gdiplus = "gdiplus.dll";
-        public const string Hhctrl = "hhctrl.ocx";
-        public const string Imm32 = "imm32.dll";
-        public const string Kernel32 = "kernel32.dll";
-        public const string Loadperf = "Loadperf.dll";
-        public const string Mscoree = "mscoree.dll";
-        public const string Clr = "clr.dll";
-        public const string Msi = "msi.dll";
-        public const string Mqrt = "mqrt.dll";
-        public const string Ntdll = "ntdll.dll";
-        public const string Ole32 = "ole32.dll";
-        public const string Oleacc = "oleacc.dll";
-        public const string Oleaut32 = "oleaut32.dll";
-        public const string Olepro32 = "olepro32.dll";
-        public const string PerfCounter = "perfcounter.dll";
-        public const string Powrprof = "Powrprof.dll";
-        public const string Psapi = "psapi.dll";
-        public const string Shell32 = "shell32.dll";
-        public const string User32 = "user32.dll";
-        public const string Uxtheme = "uxtheme.dll";
-        public const string WinMM = "winmm.dll";
-        public const string Winspool = "winspool.drv";
-        public const string Wtsapi32 = "wtsapi32.dll";
-        public const string Version = "version.dll";
-        public const string Vsassert = "vsassert.dll";
-        public const string Fxassert = "Fxassert.dll";
-        public const string Shlwapi = "shlwapi.dll";
-        public const string Crypt32 = "crypt32.dll";
-
-        // system.data specific
-        internal const string Odbc32 = "odbc32.dll";
-        internal const string SNI = "System.Data.dll";
-
-        // system.data.oracleclient specific
-        internal const string OciDll = "oci.dll";
-        internal const string OraMtsDll = "oramts.dll";
+        public const string Dnsapi = "dnsapi.dll";
```

Comment:
Can these move to [https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/Interop/Windows/Interop.Libraries.cs](https://github.com/dotnet/runtime/blob/main/src/libraries/Common/src/Interop/Windows/Interop.Libraries.cs)?

Reply:
See https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/interop-guidelines.md#file-organization

Reply:
I could.  I wasn't intending on changing things to fully comply with the interop guidelines but rather to just get closer to them.  But just moving these constants to common seems simple and reasonable enough to do.

Reply:
Fixed